content/en/partners/_index.html

```diff
@@ -10,40 +10,49 @@
-        <div id="kcspContainer"></div>
-        </div>
-        <div class="dist_section">
-        <h3><a id="dist"></a>Certified Platforms & Distributions</h3>
-        <p>This program gives end users the confidence that when they use a Certified Kubernetes product they can rely on a high level of common functionality. It gives Independent Software Vendors (ISVs) confidence that if their customer is using a Certified Kubernetes platform that their software will behave as expected.</p>
-        <p>Interested in becoming a Certified Certified Platform or Distribution? <b><a href="https://www.cncf.io/certification/software-conformance/">Learn more.</b></a></p>
-        <div id="distContainer"></div>
-        </div>
-        <div class="ktp_section">
-        <h3><a id="ktp"></a>Kubernetes Training Partners</h3>
-        <p>Individuals or corporations who are looking for specialized training that maps directly to the Certified Kubernetes Administrator (CKA) and Certified Kubernetes Application Developer (CKAD) exams can choose from a list of KTPs who have passed this rigorous qualification process. Learn more about the program <b><a href="https://www.cncf.io/certification/training/">here</a></b></p>
-        <p>Interested in becoming a Kubernetes Training Partner? Please <b><a href="https://goo.gl/forms/lwKx3U24oIumFXvM2">fill out this form.</b></a></p>
-        <div id="ktpContainer"></div>
-        </div>
-          <div  class="tech_section">
-        <h3><a id="technology"></a>Technology Partners</h3>
-        <p>Technology partners offer integrations and plugins that add features to Kubernetes applications.</p>
-        <div id="isvContainer"></div>
-        </div>
-       <div class="service_Section">
-        <h3><a id="service"></a>Service Partners</h3>
-        <p>Service Partners offer consulting or management services to help companies implement and use Kubernetes in commercial applications.</p>
-        <div id="servContainer"></div>
+          <div class="col-nav"><center><h5><b>Kubernetes Certified Service Providers</b></h5>Vetted service providers with deep experience helping enterprises successfully adopt Kubernetes.<br><br><button id="defaultIframe" class="button" onClick="updateSrc(this.id)">See KCSP Partners</button></center></div>
+          <div class="col-nav"><center><h5><b>Certified Platforms & Distributions</b></h5>Software conformance ensures that every vendor’s version of Kubernetes supports the required APIs.<br><br><button id="firstIframe" class="button" onClick="updateSrc(this.id)">See Conformance Partners</button></a></center></div>
+          <div class="col-nav"><center><h5><b>Kubernetes Training Partners</b></h5>Vetted training providers who have deep experience in cloud native technology training.<br><br><br><button id="secondIframe" class="button" onClick="updateSrc(this.id)">See KTP Partners</button></center></div>
```

Comment:
I was imagining that you would have one long embed for all 5 sections, but what you have done is probably better. "Certified Platforms & Distributions" should be "Certified Kubernetes Distributions, Hosted Platforms, and Installers"